src/open_inwoner/templates/pages/profile/appointments.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'master.html' %}
+{% load i18n tz form_tags anchor_menu_tags list_tags icon_tags grid_tags utils %}
+
+{% block content %}
+
+<h1 class="utrecht-heading-1" id="title">{% trans "Mijn afspraken" %}</h2>
+
+{% if appointments %}
+    <p class="tabled__key">{% trans "Een overzicht van uw afspraken" %}</p>
+
+    {% render_grid %}
+        {% for appointment in appointments %}
+            {% render_column start=forloop.counter_0|multiply:4 span=4 %}
+                <div class="card card--compact card--stretch appointment-info">
+                    <div class="card__body">
+                        {% render_list %}
+                            <h2 class="card__heading-2">{{ appointment.title }}</h2>
+                            {% timezone appointment.branch.timeZone %}
+                                {% list_item text=appointment.start|date:"l j F Y H:i" compact=True strong=False %}
+                            {% endtimezone %}
+                            {% list_item text=appointment.notes compact=True strong=False %}
+                            {% list_item text=appointment.branch.name caption=_("Locatie") compact=True strong=False %}
+                            {% list_item text=appointment.branch.addressCity compact=True strong=False %}
+                            {% list_item text=appointment.branch.addressLine2 compact=True strong=False %}
```

Comment:
I'm not sure if these are the right fields to display, because the documentation isn't too clear about what is stored in these fields

Reply:
Feels like a testing and acceptance problem :wink: